client/allocrunner/taskrunner/envoybootstrap_hook.go

```diff
@@ -148,3 +163,14 @@ func (h *envoyBootstrapHook) Prestart(ctx context.Context, req *interfaces.TaskP
 	resp.Done = true
 	return nil
 }
+
+func buildEnvoyAdminBind(alloc *structs.Allocation, taskName string) string {
+	port := envoyBaseAdminPort
+	for idx, task := range alloc.Job.LookupTaskGroup(alloc.TaskGroup).Tasks {
+		if task.Name == taskName {
+			port += idx
+			break
+		}
+	}
+	return fmt.Sprintf("localhost:%d", port)
```

Comment:
Why the addr instead of just the port? Is there potential for this to bite us if we make the bind address/interface configurable someday? Seems like we can't setup a test today that would break in the future.

That being said I can't think of a reason you'd bind this to anything but localhost, so this is fine.

Reply:
Yeah since this can expose tls certificates the more restricted the better IMO